protocol/src/main/java/org/fidoalliance/fdo/protocol/CryptoService.java

```diff
@@ -1154,38 +1165,56 @@ protected KeyExchangeResult getEcdhSharedSecret(byte[] message, Composite ownSta
       final byte[] encoded = ownState.getAsBytes(Const.FOURTH_KEY);
       final PKCS8EncodedKeySpec privateSpec = new PKCS8EncodedKeySpec(encoded);
       final KeyFactory factory = getKeyFactoryInstance(Const.EC_ALG_NAME);
+      final KeyAgreement keyAgreement = getKeyAgreementInstance(Const.ECDH_ALG_NAME);
       final ECPrivateKey ownKey = (ECPrivateKey) factory.generatePrivate(privateSpec);
+      final byte[] sharedSecret;
+      try {
 
-      final ECPoint w = new ECPoint(new BigInteger(1, theirX), new BigInteger(1, theirY));
-      final ECPublicKeySpec publicSpec = new ECPublicKeySpec(w, ownKey.getParams());
+        final ECPoint w = new ECPoint(new BigInteger(1, theirX), new BigInteger(1, theirY));
+        final ECPublicKeySpec publicSpec = new ECPublicKeySpec(w, ownKey.getParams());
 
-      final PublicKey theirKey = factory.generatePublic(publicSpec);
+        final PublicKey theirKey = factory.generatePublic(publicSpec);
```

Comment:
indentation correct?

Reply:
never mind, I see it now

Reply:
Yes.  Code's been moved inside of a try{} block and so get moved down one layer.